feat(changelog): dedibox-changed-start-9-m-price-drop-now-at-euro-3999 2025-06-27 (#5198)
scaleway/docs-content: changelog/june2025/2025-06-27-dedibox-changed-start-9-m-price-drop-now-at-euro-3999.mdx

---

> title: Start-9-M price drop, now at € 39.99!

Reviewer: ```suggestion
title: Start-9-M price drop, now at €39.99!
```

status: changed
date: 2025-06-27
category: bare-metal
product: dedibox
---

The AMD Ryzen CPU-based offer Start-9-M now comes with a reduced price:
* Old price: €59.99/month
* New price: €39.99/month

> Usual commitments of 12 months and 36 months can also apply making you benefit respectively of extra 5% or 15% off on the monthly fee.

Reviewer 2: ```suggestion
The usual commitments of 12 or 36 months can also apply, allowing you to benefit from an extra 5% or 15% respectively off the monthly fee.
```
